include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -341,8 +341,7 @@ struct tuple<T1, T...>
     template <typename _U1, typename... _U,
               typename = ::std::enable_if_t<
                   (sizeof...(_U) == sizeof...(T) &&
-                   oneapi::dpl::__internal::__conjunction<::std::is_constructible<T1, _U1&&>,
-                                                          ::std::is_constructible<T, _U&&>...>::value)>>
+                   std::conjunction<::std::is_constructible<T1, _U1&&>, ::std::is_constructible<T, _U&&>...>::value)>>
```

Comment:
If we are changing the lines.... lets use `std::is_constructible_v` as well.

Reply:
`std::conjunction` works with types, not values. It should be `conjunction_v<is_constructible<...>,...>`